src/site/design-elements/buttons/index.html

```diff
@@ -1,23 +1,27 @@
 <div id="buttons" class="headroom-large">
-  <h5>Buttons</h5>
+  <h2>Buttons</h2>
   <p>This is the primary button style for important calls to action. Two primary buttons cannot be used next to each other.</p>
   <div data-xrayhtml>
-    <button class="btn btn-mapzen">main call to action</button>
+    <button class="btn btn-darkpurple">primary button</button>
```

Comment:
I like that naming scheme better! Let me know which one you prefer and I can change